Gir.toml

```diff
@@ -678,6 +678,22 @@ status = "generate"
     name = "set_can_store"
     #array with size
     ignore = true
+    [[object.function]]
+    name = "request_targets"
+    #array with size
```

Comment:
What's the problem with this? We support that elsewhere just fine and we have conversion traits for this specific case :) `from_glib_none_num()`

Same for everything else below

Reply:
Then that means that my gir PR is incomplete. :'(

Reply:
Indeed :(